openerp/tools/translate.py

```diff
@@ -742,18 +742,20 @@ def get_root_view(xml_id):
                 _logger.error("name error in %s: %s", xml_name, str(exc))
                 continue
             objmodel = registry.get(obj.model)
-            if (objmodel is None or field_name not in objmodel._columns
+            if (objmodel is None or (
+                        field_name not in objmodel._columns and
+                        field_name not in objmodel._fields)
```

Comment:
Well, I think that all the old-API style columns declaration, but I'm not sure about this one.